config/initializers/new_framework_defaults_6_0.rb

```diff
@@ -37,7 +37,7 @@
 # If you send mail in the background, job workers need to have a copy of
 # MailDeliveryJob to ensure all delivery jobs are processed properly.
 # Make sure your entire app is migrated and stable on 6.0 before using this setting.
```

Comment:
I think we're stable on 6.0 (?), so maybe we can just remove this comment :D

Reply:
The file was auto generated by Rails but I can remove it yeah